projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy_print.php

```diff
@@ -160,228 +173,237 @@ function formatted_page($doctype='xhtml1.1', $externalcss=true, $title='', $lang
+				$output .= $template[0] . '@import ' . $template[5] . $this->import[ $i ] . $template[6];
 			}
 		}
-		if (!empty($this->namespace)) {
-			if (substr($this->namespace, 0, 4) === 'url(' && substr($this->namespace, -1, 1) === ')') {
-				$this->namespace = '\'' . substr($this->namespace, 4, -1) . '\'';
-				$this->parser->log('Optimised @namespace : Removed "url("', 'Information');
+		if ( ! empty( $this->namespace ) ) {
+			if ( substr( $this->namespace, 0, 4 ) === 'url(' && substr( $this->namespace, -1, 1 ) === ')' ) {
+				$this->namespace = '\'' . substr( $this->namespace, 4, -1 ) . '\'';
+				$this->parser->log( 'Optimised @namespace : Removed "url("', 'Information' );
 			}
 			$output .= $template[0] . '@namespace ' . $template[5] . $this->namespace . $template[6];
 		}
 
-		$output .= $template[13];
+		$output   .= $template[13];
 		$in_at_out = '';
-		$out = & $output;
+		$out       = & $output;
 
-		foreach ($this->tokens as $key => $token) {
-			switch ($token[0]) {
+		foreach ( $this->tokens as $key => $token ) {
+			switch ( $token[0] ) {
 				case AT_START:
-					$out .= $template[0] . $this->_htmlsp($token[1], $plain) . $template[1];
-					$out = & $in_at_out;
+					$out .= $template[0] . $this->htmlsp( $token[1], $plain ) . $template[1];
+					$out  = & $in_at_out;
 					break;
 
 				case SEL_START:
-					if ($this->parser->get_cfg('lowercase_s'))
-						$token[1] = strtolower($token[1]);
-					$out .= ( $token[1][0] !== '@') ? $template[2] . $this->_htmlsp($token[1], $plain) : $template[0] . $this->_htmlsp($token[1], $plain);
+					if ( $this->parser->get_cfg( 'lowercase_s' ) ) {
+						$token[1] = strtolower( $token[1] );
+					}
+					$out .= ( $token[1][0] !== '@' ) ? $template[2] . $this->htmlsp( $token[1], $plain ) : $template[0] . $this->htmlsp( $token[1], $plain );
 					$out .= $template[3];
 					break;
 
 				case PROPERTY:
-					if ($this->parser->get_cfg('case_properties') === 2) {
-						$token[1] = strtoupper($token[1]);
-					} elseif ($this->parser->get_cfg('case_properties') === 1) {
-						$token[1] = strtolower($token[1]);
+					if ( $this->parser->get_cfg( 'case_properties' ) === 2 ) {
+						$token[1] = strtoupper( $token[1] );
+					} elseif ( $this->parser->get_cfg( 'case_properties' ) === 1 ) {
+						$token[1] = strtolower( $token[1] );
 					}
-					$out .= $template[4] . $this->_htmlsp($token[1], $plain) . ':' . $template[5];
+					$out .= $template[4] . $this->htmlsp( $token[1], $plain ) . ':' . $template[5];
 					break;
 
 				case VALUE:
-					$out .= $this->_htmlsp($token[1], $plain);
-					if ($this->_seeknocomment($key, 1) == SEL_END && $this->parser->get_cfg('remove_last_;')) {
-						$out .= str_replace(';', '', $template[6]);
+					$out .= $this->htmlsp( $token[1], $plain );
+					if ( $this->seeknocomment( $key, 1 ) == SEL_END && $this->parser->get_cfg( 'remove_last_;' ) ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
```

Comment:
I can't see why, if `$this->seeknocomment( $key, 1 )` did return `"4"` instead of `4`, it wouldn't be a bug to consider it as being `SEL_END`.